examples/python-django/devenv.nix

```diff
@@ -0,0 +1,43 @@
+{ pkgs, config, ... }:
+
+let
+  db_user = "postgres";
+  db_host = "localhost";
+  db_port = "5432";
+  db_name = "db";
+in
+{
+  packages = [ pkgs.git pkgs.postgresql_14 ];
+
+  languages.python = {
+    enable = true;
+    package = pkgs.python310;
+    poetry.enable = true;
+  };
+
+  env = {
+    DATABASE_URL = "postgres://${db_user}@/${db_name}?host=${config.env.PGHOST}";
+    DEBUG = true;
+    SECRET_KEY = "123";
+    STATIC_ROOT = "/tmp";
+  };
+
+  services.postgres = {
+    enable = true;
+    initialScript = "CREATE USER ${db_user} SUPERUSER;";
+    initialDatabases = [{ name = db_name; }];
+    listen_addresses = db_host;
+  };
+
+  processes = {
+    runserver.exec = ''
+      devenv shell python manage.py runserver
```

Comment:
```suggestion
      python manage.py runserver
```

Reply:
There's no need to run `devenv shell` here since it's already running in the shell